Propagate OV*QuantizationConfig kwargs to nncf calls (#1179)
huggingface/optimum-intel: docs/source/openvino/optimization.mdx

)
```

> Note: `OVWeightQuantizationConfig` also accepts key word arguments not listed in its constructor. In this case such arguments will be passed directly to `nncf.compress_weights()` call. This is useful for passing additional parameters to the quantization algorithm.

Reviewer: ```suggestion
Note: `OVWeightQuantizationConfig` also accepts keyword arguments that are not listed in its constructor. In this case such arguments will be passed directly to `nncf.compress_weights()` call. This is useful for passing additional parameters to the quantization algorithm.
```
